proc_CPMG.py

```diff
@@ -18,27 +19,17 @@
     ###{{{loading in data and displaying raw data
     s = find_file(searchstr, exp_type=exp_type,
             expno=nodename, postproc=postproc, lookup=postproc_dict)
- 
-    logging.basicConfig(filename="logfile.txt")
-    stderrLogger=logging.StreamHandler()
-    stderrLogger.setFormatter(logging.Formatter(logging.BASIC_FORMAT))
-    logging.getLogger().addHandler(stderrLogger)
     nEchoes = s.get_prop('acq_params')['nEchoes']
-    #fl.next('raw data - chunking ft')
-    #fl.image(s)
     #}}}
     #{{{select and display coherence channel centered
     s.ft(['ph1'])
```

Comment:
belongs in preprocessing?

Reply:
I think there was a miscommunication here:

I said that this section (meaning the ft of the phase cycle, and anything that we're doing with `nEchoes`, since these are the parts of the code the comment refers to -- although note it would be good to do the ft along the direct dimension as well, since we're doing that)

You said "done" but the code doesn't show up as "outdated" on the top right, which means that it's hasn't changed on the server

Again, looking for the "outdated" is a fast + easy way to see whether or not the code has changed in response to a comment.

Reply:
re-read my last comment

Reply:
I don't know why this doesn't show up as outdated as I just changed it quite a bit here to implement what you asked for, can you check the actual file itself and see the changes? we can discuss today in meeting